packages/react-components/react-color-picker-preview/library/src/utils/adjustChannel.ts

```diff
@@ -0,0 +1,26 @@
+import type { ColorChannel } from '../components/ColorSlider/ColorSlider.types';
+import { MIN, HUE_MAX, MAX as COLOR_MAX } from './constants';
+import { clamp } from '@fluentui/react-utilities';
+
+export function clampValue(value: number, channel: ColorChannel = 'hue') {
+  const MAX = channel === 'hue' ? HUE_MAX : COLOR_MAX;
+  return clamp(value, MIN, MAX);
+}
+
+export type ChannelActions<T> = {
+  hue: T;
+  saturation: T;
+  value: T;
+};
+
+export function adjustChannel<T>(channel: ColorChannel, actions: ChannelActions<T>): T {
```

Comment:
I put this switch separately because we'll have more channels. But the details is being discussed with the design team